app/Checks.php

```diff
@@ -82,7 +82,8 @@ public static function postAuth()
                 Toastr::error("<a href='notifications/'>$notification->body</a>", $notification->title);
             }
 
-            if (Device::isUp()->where('last_polled', '<=', Carbon::now()->subMinutes(15))->exists()) {
+            $rrdstep = Config::get('rrd.step', 300);
+            if (Device::isUp()->where('last_polled', '<=', Carbon::now()->subSeconds($rrdstep * 2))->exists()) {
```

Comment:
Any reason to change this from 15 mins to in essence 10 mins. There was no real reason to pick 15m in the first place but interested why change it.

Reply:
Yes.

Reply:
But to answer your question, because I was changing from a flat number to a factor of rrd step, and I didn't think 3x made sense.  If someone has 1hr polling, it has to fail for 3hrs before the alert.

Maybe 3x does make sense.  Open to thoughts of course.

Reply:
I don't think many people are going to do 1 hour polling. 1 minute is more likely so this would lead to the message after 2 mins.

Either way, can you also update the actual message returned as it's hard coded to 15 minutes.

Reply:
The reason it was added is someone is doing 30 minute polling.  1hr was just round numbers.